### PR TITLE
adds logo to drawer title and updates favicon

### PR DESCRIPTION
**Pull request by vsandvold**
Based on https://github.com/Unleash/unleash/pull/100#issuecomment-212603279

### CODE REVIEW

**Comment by vsandvold** [comment]
fireshot capture 252 - strategies unleash - http___localhost_3000_ _strategies

File: src/component/app.jsx [hunk 1]
```diff
@@ -122,7 +122,10 @@ export default class App extends Component {
                         </Navigation>
                     </Header>
                     <Drawer className="mdl-color--white">
-                        <span className={[styles.drawerTitle, 'mdl-layout-title'].join(' ')}>Unleash</span>
+                        <span className={[styles.drawerTitle, 'mdl-layout-title'].join(' ')}>
+                            <img src="/public/logo.png" width="32" height="32" className={styles.drawerTitleLogo}/>
```
Comment (SimenB) [inline src/component/app.jsx]: Is there an svg available?

Comment (vsandvold) [inline src/component/app.jsx]: No, this is just the best we can do for now. The design looks great, though!

**Comment by Andersos** [comment]
Cool logo!

**Comment by vsandvold** [comment]
@sveisvei got some serious skills :-)

**Comment by sveisvei** [comment]
@vsandvold lol mockup software made  🤦‍♂️

**Pull request merged**